### PR TITLE
IA-1351 fix automation tests

### PR DESCRIPTION
**Pull request by Qi77Qi**
https://broadworkbench.atlassian.net/browse/IA-1351

Have you read [CONTRIBUTING.md](https://github.com/DataBiosphere/leonardo/blob/develop/CONTRIBUTING.md) lately? If not, do that first.

I, the developer opening this PR, do solemnly pinky swear that:

- [ ] I've documented my API changes in Swagger

In all cases:

- [ ] Get a thumbsworth of review and PO signoff if necessary
- [ ] Verify all tests go green
- [ ] Squash and merge; you can delete your branch after this
- [ ] Test this change deployed correctly and works on dev environment after deployment

### CODE REVIEW

**Review by rtitle — APPROVED** [review]
:shipit: when tests pass

**Comment by Qi77Qi** [comment]
jenkins retest

**Comment by rtitle** [comment]
Still not working I think. I see 3 errors:
- `sc is not defined` <-https://github.com/DataBiosphere/leonardo/blob/develop/automation/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/notebooks/NotebookHailSpec.scala#L48
- PySpark2 kernel no longer exists <- https://github.com/DataBiosphere/leonardo/blob/develop/automation/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/notebooks/NotebookHailSpec.scala#L74
- PySpark3 spark job failed <- https://fc-jenkins.dsp-techops.broadinstitute.org/job/leonardo-fiab-test-runner/16465/artifact/failure_screenshots/NotebookHailSpec_02-49-29-931.png

**Comment by rtitle** [comment]
Trying some fixes...

File: automation/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/ClusterFixtureSpec.scala [hunk 1]
```diff
@@ -89,13 +97,14 @@ abstract class ClusterFixtureSpec extends fixture.FreeSpec with BeforeAndAfterAl
 
   override def afterAll(): Unit = {
     logger.info("afterAll")
-    if (!debug) {
+    if (debug)
+      super.afterAll()
```
Comment (rtitle) [inline automation/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/ClusterFixtureSpec.scala]: Is this right? We'd want to call `afterAll` if NOT in debug mode, right?

File: automation/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/notebooks/NotebookCustomizationSpec.scala [hunk 2]
```diff
@@ -84,7 +84,7 @@ final class NotebookCustomizationSpec extends GPAllocFixtureSpec with ParallelTe
               serverExt.get should include("jupyter_nbextensions_configurator  enabled")
 
               // Exercise the translate extension
-              notebookPage.translateMarkup("Hello") should include("Bonjour")
+              notebookPage.translateMarkup("Hello") should include("Salut")
```
Comment (rtitle) [inline automation/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/notebooks/NotebookCustomizationSpec.scala:87]: Au revoir

**Comment by rtitle** [comment]
Latest change looks good, 1 question in `ClusterFixtureSpec`

File: automation/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/ClusterFixtureSpec.scala (hunk 1 above)
Comment (Qi77Qi) [reply automation/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/ClusterFixtureSpec.scala]: no...just fixed it :)

**Comment by rtitle** [comment]
:shipit:

**Pull request merged**